test/helpers/kubectl.go

```diff
@@ -1271,101 +1271,86 @@ type ResourceLifeCycleAction string
 // imported before the timeout is
 // exceeded.
 func (kub *Kubectl) CiliumPolicyAction(namespace, filepath string, action ResourceLifeCycleAction, timeout int64) (string, error) {
-	revisions := map[string]int{}
+	numNodes := kub.GetNumNodes()
 
+	// Test filter: https://jqplay.org/s/J3epZ41h-c
+	jqFilter := fmt.Sprintf(
```

Comment:
This filter only makes sense for the apply and create action, not the delete action.

Reply:
Done: 
https://github.com/cilium/cilium/blob/6fc3ac5e3c0e79d55e8db8b9e01c0eb345a69e2c/test/helpers/kubectl.go#L1257-L1260